sdk/core/core/test/curl_easy_perform_poc.c

```diff
@@ -3,39 +3,32 @@
 
 #include <stdio.h>
 
-#include <curl/curl.h>
-
-#include <_az_cfg.h>
-
-int main(int _, char ** argv) {
-  CURL * curl;
-  curl = curl_easy_init();
-  int result;
-
-  FILE * f;
-  f = fopen("here.jpg", "wb");
-
-  printf("\n1...");
-
-  curl_easy_setopt(
-      curl,
-      CURLOPT_URL,
-      "http://www.laboratoons.com/wp-content/uploads/2018/11/Alternativas-A.jpg");
-  curl_easy_setopt(curl, CURLOPT_WRITEDATA, f);
-  curl_easy_setopt(curl, CURLOPT_FAILONERROR, 1l);
-
-  result = curl_easy_perform(curl);
-
-  if (result == CURLE_OK) {
-    printf("\n Done.");
-  } else {
-    printf("\n Error: %s\n", curl_easy_strerror(result));
-  }
-
-  // clean up
-  fclose(f);
-  curl_easy_cleanup(curl);
-
-  printf("Hello world");
-  return _;
+#include <az_write_span_iter.h>
+#include <az_http_request.h>
+#include <az_pair.h>
+#include <az_json_read.h>
```

Comment:
Run formatting, it will sort

Reply:
I would strongly discourage to sort headers. This type of "formatting" may change compilation order which can be very important. Please, don't sort includes.